app/templates/index.hbs

```diff
@@ -0,0 +1,5 @@
+<h1>REMINDERS</h1>
+
+{{#each model as | reminder |}}
+  <ul><li class='spec-reminder-item'>{{reminder}}</li></ul>
```

Comment:
As a follow up - this is ok if you want to treat each reminder as a UL< although I think it makes more sense to treat each reminder as an article with a title (say maybe h2), date (p) and notes (p).